src/openrct2/drawing/drawing.h

```diff
@@ -363,6 +363,8 @@ void shorten_path(utf8 *buffer, size_t bufferSize, const utf8 *path, sint32 avai
 void scrolling_text_initialise_bitmaps();
 sint32 scrolling_text_setup(rct_string_id stringId, uint16 scroll, uint16 scrollingMode);
 
+void FASTCALL gfx_get_sprite_size(sint32 image_id, sint32* width, sint32* height);
```

Comment:
I'll leave that to @IntelOrca and @janisozaur . I don't know enough of this.

But you really should try to stick to one thing per PR. Doing too much in one PR complicates matters a lot.

For now, just wait until they give an answer. If they like it, stick it in a separate PR.